cfme/tests/control/test_alerts.py

```diff
@@ -128,7 +129,7 @@ def configure_fleecing(appliance, provider, vm, vddk_url):
 
 @pytest.fixture
 def setup_for_alerts(alert_profile_collection, action_collection, policy_collection,
```

Comment:
Could we have some sort of a function header here.   I know its not your code, but somethings that explains what this things is supposed to accomplish with a little more detail than what is in the name.

Reply:
I'll add something in